src/main/java/org/apache/xmlbeans/XmlCursor.java

```diff
@@ -1442,6 +1460,10 @@ public final XmlCursor createCursor() {
          * @param c the cursor to be moved
          * @return the given cursor moved to this bookmark
          */
+        // TODO Problematic API for an AutoCloseable XmlCursor, it returns the same
```

Comment:
JIRA is used for tracking issues - if you don't intend to do anything about this - can you log a JIRA and replace the TODO comment with a comment that provides a link to the JIRA?